eslint.config.mjs

```diff
@@ -22,12 +22,6 @@ const config = createConfig([
       // design system react
       'packages/design-system-react/src/components/Icon/icons/*.tsx',
       'packages/design-system-react/src/components/Icon/icons/index.ts',
-      'packages/design-system-react/src/components/AvatarFavicon/AvatarFavicon.test.tsx',
-      'packages/design-system-react/src/components/AvatarGroup/AvatarGroup.stories.tsx',
-      'packages/design-system-react/src/components/AvatarGroup/AvatarGroup.test.tsx',
-      'packages/design-system-react/src/components/AvatarGroup/AvatarGroup.tsx',
-      'packages/design-system-react/src/components/AvatarNetwork/AvatarNetwork.test.tsx',
-      'packages/design-system-react/src/components/AvatarToken/AvatarToken.test.tsx',
```

Comment:
Removing avatars from file ignores in eslint config, this also enables automatic linting fixes like import order